samples/iolib/src/main/cpp/player/SampleBuffer.cpp

```diff
@@ -34,7 +34,9 @@ void SampleBuffer::loadSampleData(parselib::WavStreamReader* reader) {
 }
 
 void SampleBuffer::unloadSampleData() {
-    delete[] mSampleData;
+    if (mSampleData != nullptr) {
+        delete[] mSampleData;
```

Comment:
It would be nice to clear the pointer in case this gets called again.

    mSampleData = nullptr;

Reply:
Done. Thnaks.